lesson9/index.js

```diff
@@ -0,0 +1,40 @@
+const ul = document.getElementById('js-lists');
+
+const listData = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+]
+
+async function promise() {
+    return new Promise(resolve => {
+        const lodingImage = document.createElement('img');
+        lodingImage.src = './loading-circle.gif';
+        ul.appendChild(lodingImage);
+        setTimeout(() => {
+            resolve(listData);
+            lodingImage.remove();
+        }, 3000);
+    });
+}
+
+async function asyncCall() {
```

Comment:
asyncCall as function name is not good. Express what the function is doing. This is a bit too abstract

Reply:
I understand.
I named `createLists`.
commit: c75f2c9